genesis/process/metaGenesisBlockCreator.go

```diff
@@ -500,13 +501,13 @@ func createProcessorsForMetaGenesisBlock(arg ArgsGenesisBlockCreator, enableEpoc
 		BalanceComputation:                   disabledBalanceComputationHandler,
 		EconomicsFee:                         genesisFeeHandler,
 		TxTypeHandler:                        txTypeHandler,
-		BlockGasAndFeesReCheckEnableEpoch:    enableEpochs.BlockGasAndFeesReCheckEnableEpoch,
+		BlockGasAndFeesReCheckEnableEpoch:    enableEpochsConfig.BlockGasAndFeesReCheckEnableEpoch,
```

Comment:
I guess this will be refactored in a future PR

Reply:
exactly